### PR TITLE
Return changes in gui format

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice/8/
Test PASSed.

File: params/params.go [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+	Args     []interface{} `json:"args"`
+	Id       string        `json:"id"`
+	Requires []string      `json:"requires"`
+	Method   string        `json:"method"`
+}
 
 // ChangesResponse contains the results of parsing a bundle into a list of
 // changes.
 type ChangesResponse struct {
 	// TODO This should be an API specific type
```
Comment (mhilton) [inline params/params.go]: Is this TODO done?

**Comment by mhilton** [comment]
:+1:

File: params/params.go (hunk 1 above)
Comment (frankban) [inline params/params.go]: Yes it is.

**Comment by frankban** [comment]
👍  thank you!

**Comment by makyo** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice/9/
Test PASSed.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/bundleservice-merge/2

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/bundleservice-merge/3

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice/10/
Test PASSed.

**Comment by makyo** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/bundleservice-merge/4

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/bundleservice-merge/5

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice/11/
Test PASSed.

**Comment by makyo** [comment]
Third time's the charm? :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Pull request merged**